otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/akka/http/UpdateHttpRouteWrapper.scala

```diff
@@ -0,0 +1,33 @@
+package io.scalac.mesmer.otelextension.instrumentations.akka.http
+
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.model.HttpResponse
+import io.opentelemetry.context.Context
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpRouteHolder
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpRouteSource
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.Using
+
+final class UpdateHttpRouteWrapper(inner: HttpRequest => Future[HttpResponse])(implicit
```

Comment:
Can this simply be called "HttpRouteUpdater"?

Reply:
I think that `Wrapper` is a valid name, as it reflects what this class actually does 🙂